### PR TITLE
HEEDLS-NONE Assessment page diagnostic tests

### PR DESCRIPTION
**Pull request by kwunkiuma**
Add some tests to compare results from the service class to results from stored procedures.

### CODE REVIEW

File: DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace DigitalLearningSolutions.Data.Models.DiagnosticAssessment
+{
+    public class OldTutorial
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs:3]: Don't we already have a class like this? Or am I thinking of one @SChandarana has added in an open PR?

**Review by FridaTveit — COMMENTED** [review]
One question, otherwise looks good 🙂

File: DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs (hunk 1 above)
Comment (SChandarana) [reply DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs:3]: Yep I have "OldSectionTutorial.cs" in PR #110 which is the same as this

Comment (FridaTveit) [reply DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs:3]: In that case whichever one of you merges last should rebase and use the existing class. I'll leave it up to you to coordinate that 🙂 @SChandarana I think OldTutorial might be a better name 🙂

Comment (kwunkiuma) [reply DigitalLearningSolutions.Data/Models/DiagnosticAssessment/OldTutorial.cs:3]: We've sorted out the order for this, are there any other comments for this PR?

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**